### PR TITLE
Write ObeysRule

### PR DESCRIPTION
**Pull request by mint-thompson**
Completes task [CIMPL-515](https://standardhealthrecord.atlassian.net/browse/CIMPL-515).

Instances of `ExportableObeysRule` are found in the `rules` of an `ExportableProfile` or `ExportableExtension` instance. So, not a whole lot of work is needed to make sure they will be exported when found. The only slight nuance is that a path on the root element is given as no path, instead of the `.` path used in other rules.

### CODE REVIEW

**Review by cmoesel — DISMISSED** [review]
Yep.  Looks good!

**Review by ngfreiter — APPROVED** [review]
Looks good!

File: test/exportable/ExportableExtension.test.ts [hunk 1]
```diff
@@ -98,16 +99,22 @@ describe('ExportableExtension', () => {
     input.rules.push(flagRule);
 
     const bindingRule = new ExportableBindingRule('value[x]');
-    bindingRule.valueSet = 'http://example.org/ValueSet/Foo';
+    bindingRule.
+      = 'http://example.org/ValueSet/Foo';
```
Comment (jafeltra) [inline test/exportable/ExportableExtension.test.ts:103]:
```suggestion
    bindingRule.valueSet = 'http://example.org/ValueSet/Foo';
```

Comment (cmoesel) [reply test/exportable/ExportableExtension.test.ts:103]: Thank you for fixing that nasty bad good for nothing merge commit.

**Review by cmoesel — APPROVED** [review]
Re-approving!

**Pull request merged**